doc/source/methods/Anchors.ipynb

```diff
@@ -49,6 +49,38 @@
     "As highlighted by the above example, an anchor explanation consists of *if-then rules*, called the anchors, which sufficiently guarantee the explanation locally and try to maximize the area for which the explanation holds. This means that as long as the anchor holds, the prediction should remain the same regardless of the values of the features not present in the anchor. Going back to the sentiment example: as long as *not good* is present, the sentiment is negative, regardless of the other words in the movie review."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use-case insights"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As previously mentioned, the power of the Anchors over other local explanations methods comes from the objective formulation which is to maximize the **coverage** under the **precision** constraints. For a more intutive understanding of what the method tries to achieve we will loosely define the two concepts.\n",
+    "\n",
+    "**Precision** represents the probability of receiving the same classification label of the explained input if we query the model on other instances that satisfy the anchor predicates. The expected precision range is the interval $[t, 1]$, where t is the user-specified precision threshold.\n",
+    "\n",
+    "**Coverage** represents the proportion of the population which satisfy the anchor predicates. It is a positive number $\\le 1$, where a value of $1$ corresponds to the empty anchor.\n",
+    "\n",
+    "There are some edge cases that a practitioner should be aware:\n",
+    "\n",
+    "- An anchor with many predicates and a small coverage might indicate that the explained input lies near the decision boundary. Many more predicates are needed to ensure that an instance keeps the predicted label since minor perturbations may push the prediction to another class.\n",
+    "- An empty anchor with a coverage of 1 indicates that there is no salient subset of features that is necessary for the prediction to hold. In other words, with high probability (as measured by the precision), the predicted class of the data point does not change regardless of the perturbations applied to it."
```

Comment:
- Perhaps add that this is likely to occur if the data set is unbalanced
- Also maybe link to the FAQs on this